src/components/Reports/ReportForm.tsx

```diff
@@ -23,6 +23,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginLeft: theme.spacing(3),
     borderBottom: `solid 1px ${theme.palette.TwClrBrdrTertiary}`,
   },
+  textfieldDisplay: {
+    '& p': {
+      whiteSpace: 'pre-line',
+    },
+  },
```

Comment:
Interesting.. wondering if this should be part of the default css in web-components - the use-case seems legit for all textarea values in display mode.
Thoughts @charlie-tf @constanzauanini ?

Reply:
Oh you're right. That might be better than sprinkling this custom css here and there. Are there any use cases where we'd want to collapse all the newlines? I could add it as a prop to `Textfield` 🤔

Reply:
Yes, we could add it to the textfield (textarea type) component